modules/blocks/verify.js

```diff
@@ -41,6 +41,9 @@ function Verify (logger, block, transaction, db) {
  * @return {Object}   cb.err Error if occurred
  */
 __private.checkTransaction = function (block, transaction, cb) {
+	// TODO: restore transaction verifing
```

Comment:
Add trello card in todo

Reply:
https://trello.com/c/2jF7cnad/115-restore-transactions-verifing